tests/python/rest_api/test_tasks.py

```diff
@@ -1825,7 +1825,7 @@ def test_check_import_cache_after_previous_interrupted_upload(self, tasks_with_s
             url, filename, meta=params, logger=self.client.logger.debug
         )
         number_of_files = 1
-        sleep(30)  # wait when the cleaning job from rq worker will be started
+        sleep(5)  # wait when the cleaning job from rq worker will be started
```

Comment:
Why do we need to sleep only 5 seconds here?
How about changing the number of iterations in `for _ in range(12):` too?

Reply:
I suppose this value should be the same as `RQ_SCHEDULER_CHECK_INTERVAL + IMPORT_CACHE_CLEAN_DELAY`, it also seems that we can reduce `RQ_SCHEDULER_CHECK_INTERVAL` to 5-10s

Reply:
@Marishka17 please correct me if I'm wrong